### PR TITLE
rtmp-services: Update Switchboard Live servers

### PR DESCRIPTION
**Pull request by CiriousJoker**
### Description
Update server list for Switchboard Live

### Motivation and Context
The current list just wasn't up to date anymore.

### How Has This Been Tested?

Tested on Windows 10

### Types of changes

- New feature (non-breaking change which adds functionality)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.

### CODE REVIEW

**Comment by CiriousJoker** [comment]
@notr1ch @dodgepong

**Comment by Fenrirthviti** [comment]
You still have not properly filled out the PR template. Also, comments to ping members of the team are unnecessary. You should have just force pushed the branch for the previous PR. Please, in the future make sure you read our process, understand what is required, and follow it appropriately. There was no reason to create a new PR for a simple change like this. It causes unnecessary emails to be sent to people who are following the repo. If you were not sure how to do what was being asked, just ask for help.

**Comment by CiriousJoker** [comment]
I'm sorry. I tried to create as little problems as possible and ended up making it worse :(
I've read the guidelines now and updated the PR accordingly.

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -1262,33 +1262,72 @@
             }
         },
         {
-            "name": "Switchboard Live (Joicaster)",
+            "name": "Switchboard Live",
+            "alt_names": ["Switchboard Live (Joicaster)"],
+            "common": true,
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json:1267]: This is not a common service, this line should be removed.

**Comment by jp9000** [comment]
Could not push to your fork (likely due to it being on your fork's master branch), so manually merging

**Pull request closed without merging**